Addons/NetworkZones/NetworkZones_MySql.cs

```diff
@@ -0,0 +1,107 @@
+
+using UnityEngine;
+using UnityEngine.Networking;
+using System;
+using System.IO;
+using System.Linq;
+using System.Collections.Generic;
+using System.Data;
+using MySql.Data;								// From MySql.Data.dll in Plugins folder
+using MySql.Data.MySqlClient;                   // From MySql.Data.dll in Plugins folder
+
+
+using SqlParameter = MySql.Data.MySqlClient.MySqlParameter;
+
+
+/// <summary>
+/// Database class for mysql
+/// Port of the sqlite database class from ummorpg
+/// </summary>
+public partial class Database
+{
+
+    static void Initialize_Zone()
+    {
+        ExecuteNonQueryMySql(@"
+        CREATE TABLE IF NOT EXISTS character_scene (
+            `character` VARCHAR(16) NOT NULL, 
+            scenepath VARCHAR(255) NOT NULL,
+            PRIMARY KEY(`character`)
+            ) CHARACTER SET=utf8mb4");
+
+
+        ExecuteNonQueryMySql(@"
+        CREATE TABLE IF NOT EXISTS zones_online (
+            online VARCHAR(255) NOT NULL
+        ) CHARACTER SET=utf8mb4");
+    }
+
+
+    public static bool IsCharacterOnlineAnywhere(string characterName)
+    {
+        object obj = ExecuteScalarMySql("SELECT online FROM characters WHERE name=@name", new SqlParameter("@name", characterName));
+        if (obj != null)
+        {
+            string online = (string)obj;
```

Comment:
online is not a string in my addon,  I changed it to a TIMESTAMP,  this will throw an invalid cast exception.